props/blaster.h

```diff
@@ -93,9 +93,10 @@ class Blaster : public PROP_INHERIT_PREFIX PropBase {
     }
   }
 
-  bool auto_firing_ = false;
   int shots_fired_ = 0;
+  bool auto_firing_ = false;
```

Comment:
What's the point of moving this one line?

Reply:
I just grouped the bools together I guess.